js/interface.js

```diff
@@ -75,6 +75,14 @@ $('#show-full-name-field').on('click', function() {
   $('.first-last-names-holder').addClass('hidden');
 });
 
+$('#chat-contacts-settings').on('shown.bs.tab', function () {
+  dataSourceProvider.emit('show-widget');
```

Comment:
This would match better with what `Fliplet.Widget.autosize()` actually does.

```suggestion
  dataSourceProvider.emit('widget-autosize');
```